src/cluster_legacy.c

```diff
@@ -2929,8 +3121,11 @@ int clusterProcessPacket(clusterLink *link) {
                 /* If the reply has a non matching node ID we
                  * disconnect this node and set it as not having an associated
                  * address. */
-                serverLog(LL_DEBUG,"PONG contains mismatching sender ID. About node %.40s added %d ms ago, having flags %d",
+                serverLog(LL_DEBUG,
+                    "PONG contains mismatching sender ID. About node %.40s (%s) in shard %.40s added %d ms ago, having flags %d",
                     link->node->name,
+                    link->node->human_nodename,
```

Comment:
What's the default value for human nodename? I'm thinking that if it hasn't been set explicitly, it may be nonsense and not good to display it in log messages like this..?

Reply:
It's empty by default. In a past PR we decided it's better to just show an empty () instead of having if/else everywhere. The alternative was we could also have a single node identifier that is cached in the cluster_node that is printed here.

Reply:
OK, if it's printed like that in other places.